src/server/replica.cc

```diff
@@ -1023,7 +1030,21 @@ Replica::Info Replica::GetInfo() const {
     res.full_sync_done = (state_mask_.load() & R_SYNC_OK);
     res.master_last_io_sec = (ProactorBase::GetMonotonicTimeNs() - last_io_time) / 1000000000UL;
     return res;
-  });
+  };
+
+  if (ABSL_PREDICT_TRUE(Sock()))
```

Comment:
This is definitely not on the hot path, so no need for micro optimizations such as ABSL_PREDICT_TRUE :)

Reply:
lol I just had to ;)